app/jobs/autoscaling.py

```diff
@@ -23,14 +25,22 @@
 SCALING_METRIC_TAG_PREFIX = "IDSeqScalingData_"
 SCALING_METRIC_TAG_FORMAT = SCALING_METRIC_TAG_PREFIX + "{environment}"
 
-
 # A value recorded this many minutes earlier will be ignored
 EXPIRATION_PERIOD_MINUTES = 60
 
-
 DEBUG = False
 
 
+## constants specific to draining / safe scale-down:
+DRAINING_TAG = "draining"
+MAX_REFRESH_INTERVAL = 900
+# MAX_REFRESH_INTERVAL must be the same as MAX_INTERVAL_BETWEEN_DESCRIBE_INSTANCES in idseq-dag.
```

Comment:
May be just slightly longer would be better than exactly the same.

Reply:
I think it is probably okay to shorten MAX_REFRESH_INTERVAL in idseq-dag to 10 minutes.    We could also make it a system parameter in the DAG json just like max_concurrency, so it would be available from the same source in both places.  There should probably be a special section for system (infra) parameters in the dag config as opposed to parameters than affect the results.

Reply:
Agree - I changed MAX_REFRESH_INTERVAL to 1200. This may seem like too big an increase but I want to be safe with respect to synchronization lag; we can always revise it downwards later. I suppose scaling down 5 minutes earlier or later doesn't make such a huge different in cost.

Reducing MAX_INTERVAL_BETWEEN_DESCRIBE_INSTANCES seems like a good idea, but I think it should be a separate change.